graphics.cpp

```diff
@@ -51,8 +56,17 @@ void Graphics::ChangeWindowColor() {
 
 void Graphics::DrawNextFrame() {
   // Draw the next frame to the screen.
+  if (number_of_frames_drawn_ % kFramesPerFpsCheck == 0 &&
+      number_of_frames_drawn_ > 0) {
+    int current_time = SDL_GetTicks();
+    int elapsed_time = current_time - last_fps_check_;
+    float fps =
+        static_cast<float>(kFramesPerFpsCheck) / (elapsed_time / kMsPerSecond);
+    std::cout << "FPS: " << fps << std::endl;
+    last_fps_check_ = SDL_GetTicks();
+  }
+  number_of_frames_drawn_++;
```

Comment:
This is a good candidate for being refactored into a function. It could be a private function called "CalculateFps()" or something like that. That way the DrawNextFrame function isn't so ugly.